package.json

```diff
@@ -65,13 +65,13 @@
     "cross-env": "^5.2.0",
```

Comment:
@Camwyn Can you check these changes out? Not sure if we should be going back in versions here